src/ra_server.erl

```diff
@@ -74,13 +74,20 @@
 -type command_type() :: '$usr' | '$ra_query' | '$ra_join' | '$ra_leave' |
                         '$ra_cluster_change' | '$ra_cluster'.
 
--type command_meta() :: #{from := maybe(from())}.
+-type command_meta() :: #{from => from(),
+                          ts := integer()}.
 
 -type command_correlation() :: integer() | reference().
 
+-define(AFTER_LOG_APPEND, ala).
```

Comment:
Where are this defines used?

Reply:
good point, they were not meant to be there :)

Reply:
updated